cfy_manager/components/rabbitmq_status_reporter/rabbitmq_status_reporter.py

```diff
@@ -0,0 +1,28 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from ..service_names import RABBITMQ
+from ..statusreporter.status_reporter import StatusReporter
+
+from ...config import config
+
+
+class RabbitmqStatusReporter(StatusReporter):
+    def __init__(self, skip_installation):
+        super(RabbitmqStatusReporter, self).__init__(skip_installation,
+                                                     'rabbitmq_reporter')
+
+    def _should_install(self):
+        return config[RABBITMQ]['join_cluster']
```

Comment:
This is incorrect. join_cluster will only be set (optionally) on the second and subsequent cluster members.
It doesn't need to be set even then, as the node can be installed then manually joined (and this will like be the case in an image).
Also, join_cluster is a string.
The best check is probably something like return len(config[RABBITMQ]['cluster_members']) > 0 or something like that.